lib/canary/plugs.ex

```diff
@@ -222,26 +225,37 @@ defmodule Canary.Plugs do
   defp fetch_resource(conn, opts) do
     repo = Application.get_env(:canary, :repo)
 
+    id_name = get_resource_id_name(conn, opts)
```

Comment:
Sorry mate, I was too quick searching and replacing, I fully agree seeing as the variable on 228 isn't the name of the id parameter it's the actual id :see_no_evil: 

Let me fix it real quick.